GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationSummaryViewModel.cs

```diff
@@ -0,0 +1,69 @@
+﻿using GalaxyZooTouchTable.Models;
+using GalaxyZooTouchTable.Utility;
+using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+
+namespace GalaxyZooTouchTable.ViewModels
+{
+    public class ClassificationSummaryViewModel : ViewModelBase
+    {
+        static Random random = new Random();
+        public NotificationsViewModel Notifications { get; private set; }
+
+        public ICommand RandomGalaxy { get; private set; }
+        public ICommand ChooseAnotherGalaxy { get; private set; }
+        public event Action RandomGalaxyDelegate = delegate { };
+        public event Action ChooseAnotherGalaxyDelegate = delegate { };
+        public event Action<TableSubject> DropSubjectDelegate = delegate { };
+
+        readonly List<string> SummaryStrings = new List<string>
+        {
+            "Nice work! You just science'd.",
+            "Thanks for your help!",
+            "You're a real scientist!",
+            "Don't stop now!",
+            "You can do this at home on zooniverse.org."
+        };
```

Comment:
Ohhhh I see, we randomly select between these for a message upon a classification happening.

Reply:
Yeah, in an earlier commit this was done in the `App.xaml` file, but I moved it to the class using the string so tests would pass. I think it makes more sense at the class-level as well.